aws-go-console-slack-notification/Makefile

```diff
@@ -0,0 +1,5 @@
+build::
+	GOOS=linux GOARCH=amd64 go build -o ./handler/dist/handler ./handler/handler.go
+	zip -j ./handler/dist/handler.zip ./handler/dist/handler
+dep::
+	dep ensure -v
```

Comment:
Are you able to update this to go modules? I don't think we want to put out any more content referencing dep. It's considered a dead/stale tool by the community from what I understand.

Reply:
Yep, done.